packages/cli/lib/commonOpts.js

```diff
@@ -6,56 +6,59 @@ const {
   DEFAULT_MODE,
   Mode,
 } = require('@hubspot/cli-lib');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+
+const i18nKey = 'cli.lib.commonOpts';
 const { LOG_LEVEL } = Logger;
 
 const addAccountOptions = program =>
   program.option('portal', {
     alias: ['p', 'account', 'a'],
-    describe: 'HubSpot portal id or name from config',
+    describe: i18n(`${i18nKey}.options.portal.describe`),
     type: 'string',
   });
 
 const addConfigOptions = yargs =>
   yargs.option('config', {
     alias: 'c',
-    describe: 'path to a config file',
+    describe: i18n(`${i18nKey}.options.config.describe`),
     type: 'string',
   });
 
 const addOverwriteOptions = yargs =>
   yargs.option('overwrite', {
     alias: 'o',
-    describe: 'overwrite existing files',
+    describe: i18n(`${i18nKey}.options.overwrite.describe`),
     type: 'boolean',
     default: false,
   });
 
 const addModeOptions = (yargs, { read, write }) => {
   const modes = `<${Object.values(Mode).join(' | ')}>`;
-  const help = read
-    ? `read from ${modes}`
-    : write
-    ? `write to ${modes}`
-    : `${modes}`;
 
   return yargs.option('mode', {
     alias: 'm',
-    describe: help,
+    describe: i18n(
+      `${i18nKey}.options.modes.describe${
```

Comment:
missing a period here after "describe".

Side note: the error logging for missing keys is very useful! I think I noticed a bug in it though. Here's the error I got that helped me find the missing period:
```
[ERROR] Unable to access language property: en.cli.lib.commonOpts.options.modes.describewrite. Failed to access prop "modes".
```

I think it should say: `failed to access prop "describewrite"`

Reply:
Thanks for testing and point this out!

It looks like there were two issues...
1. The missing period after `describe`
2. The prop in `en.lyaml` was `mode` not `modes`, which led to the other issue.

Reply:
No prob! And it sounds like you can ignore my second point about the missing prop message since "modes" was the right thing for it to log.